lib/browser.js

```diff
@@ -42,118 +33,77 @@ class Browser {
   }
 
   init () {
-    this.collection.add(this)
+    this.log.info(`Connected on socket ${this.socket.id} with id ${this.id}`)
 
     this.bindSocketEvents(this.socket)
-
-    this.log.info('Connected on socket %s with id %s', this.socket.id, this.id)
-
-    // TODO(vojta): move to collection
-    this.emitter.emit('browsers_change', this.collection)
-
+    this.collection.add(this)
+    this.emitter.emit('browsers_change', this.collection) // TODO(vojta): move to collection
```

Comment:
Please remove ancient TODO.

Reply:
I'm planning to take a deeper look into it, but outside of this pull request - it's why I needed removed it. 

Please take a look here:
https://github.com/karma-runner/karma/blob/9bd7e1d506e0fa697b9ceb9df149852ddd10b947/lib/browser_collection.js#L12-L15

`browsers_change` is actually emitted in collection - I'm planning to check if it's enough to remove this whole line.

Reply:
ok thx